README.md

```diff
@@ -4,43 +4,207 @@ FoundationDB is a distributed database designed to handle large volumes of struc
-  nested records, including the ability to define indexes against
-  such nested structures.
-* **Queries** - The Record Layer does not provide a query language, however
-  it provides query APIs with the ability to scan, filter, and sort
-  across one or more record types, and a query planner capable of
-  automatic selection of indexes.
-* **Many record stores, shared schema** - The Record Layer provides the
-  ability to support many discrete record store instances, all with
-  a shared (and evolving) schema. For example, rather than modeling a
-  single database in which to store all users' data, each user can be
-  given their own record store, perhaps sharded across different FDB
-  cluster instances.
-* **Very light weight** - The Record layer is designed to be used in a
-  large, distributed, stateless environment. The time between opening
-  a store and the first query is intended to be measured in milliseconds.
-* **Extensible** - New index types and custom index key expressions
-  may be dynamically incorporated into a record store.
-
-The Record Layer may be used directly or provides an excellent foundational
-layer on which more complex systems can be constructed.
+# FoundationDB Record Layer (FRL)
+
+FRL provides a **relational database with SQL support** built on top of FoundationDB, featuring:
+
+* **SQL Database** - SQL support with JDBC connectivity for defining schemas,
+  querying data, and managing tables using familiar SQL syntax. The SQL API is under
+  active development with frequent enhancements.
+* **Advanced Data Types** - Beyond standard SQL types (STRING, INTEGER, FLOAT, BOOLEAN),
+  FRL supports:
+  * **Nested Structures** - User-defined struct types that can be nested arbitrarily deep
+  * **Arrays** - Collections of primitives or complex types
+  * **Vectors** - Fixed-dimension numerical vectors for ML embeddings and similarity search
+* **Schema Templates** - Reusable schema definitions that enable multi-tenant architectures
+  where each tenant gets their own database instance with a shared, evolvable schema.
+* **Intelligent Query Planning** - Automatic index selection and query optimization with
+  support for JOINs, aggregations (COUNT, SUM, etc.), GROUP BY, and ORDER BY.
+  Queries are efficiently executed using index-backed operations without in-memory sorting.
+* **Indexes** - Rich indexing capabilities including value indexes, rank indexes, aggregate
+  indexes, and indexes on nested fields. Indexes are materialized views that update incrementally.
+* **Scalable Architecture** - Designed for distributed, stateless environments with
+  millisecond-level store initialization. Perfect for applications managing thousands
+  of discrete database instances.
+* **ACID Transactions** - Full transactional semantics inherited from FoundationDB,
+  with support for continuations for efficiently paging through large result sets.
+
+## Quick Start with SQL
+
+```java
```

Comment:
Rather than having direct documentation here, we may want to consider updating the `examples` subproject with a SQL-based path. One advantage of that is that we compile and test that project as part of PRB, so it's more hardened against bit rot. But this is probably fine for now

Reply:
I kind of like having a little bit of code in the original page, before people start diving in. 
Good point about the examples subproject though, looks like it's similar to the yamsql docs files, but targeted towards Java code.
Probably not something for this PR, but good to know, thanks!